TypeCobol/Compiler/Text/CobolChar.cs

```diff
@@ -22,6 +22,11 @@ public static bool IsCobolWordChar(char chr)
             return isCobolWordChar[(byte)chr];
         }
 
+        public static bool IsCobolAlphabeticChar(char chr)
+        {
+            return ('A' <= chr && chr <= 'Z') || ('a' <= chr && chr <= 'z');
```

Comment:
char.IsLetter ?

Reply:
No because according to the doc https://docs.microsoft.com/fr-fr/dotnet/api/system.char.isletter?view=net-6.0#system-char-isletter(system-char) `IsLetter` is true for all Unicode letters. This includes `à`, `é` ...